src/Sylius/Bundle/ShopBundle/Twig/Component/Account/ProfileUpdateComponent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Account;
+
+use Sylius\Component\Core\Model\Customer;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentWithFormTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent]
+class ProfileUpdateComponent
```

Comment:
In previous PR you've created component: https://github.com/Sylius/Sylius/blob/bootstrap-shop/src/Sylius/Bundle/ShopBundle/Twig/Component/Register/FormComponent.php 

And it has same logic. Can we use that here also? 🤔 Maybe move it to `.../Component/Account/Common/FormComponent`